tock/tock/templates/hours/timecard_form.html

```diff
@@ -203,7 +189,7 @@ <h3 class="usa-alert-heading">{{ message.extra_tags }}</h3>
   function toggleNotesField(selectBox) {
     var $fieldset = $(selectBox).parents('.entry-project'),
         $selected = $(selectBox).find(':selected'),
-        $notes = $fieldset.find('#entry-notes'),
+        $notes = $fieldset.find('.entry-note-field'),
```

Comment:
Not that it probably matters for such a small number of selectors, but the older version with ID would be quicker, wouldn't it?

This might also be a good time to refactor away our jQuery dependency, or at least this part of it. Or maybe a separate issue. No strong feelings.

Reply:
Yeah, I'd prefer to handle reworking the Tock JS in another issue. Ideally we can extract _all_ of it from the templates, remove jquery, and simplify across the board. 

[Help Needed] 😄 

I'll clean this up just a bit more to remove the other instances of duplicated ID values.